common/src/main/resources/db/changelog/v2021/support_default_since.sql

```diff
@@ -0,0 +1,8 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset lsharshar:support_default_since failOnError:true
+
+ALTER TABLE job ADD since_source VARCHAR(256);
```

Comment:
Give it a default of USER and make this field NOT NULL - makes life much easier.